packages/autocomplete-shared/src/safelyRunOnBrowser.ts

```diff
@@ -0,0 +1,24 @@
+type BrowserCallback<TReturn> = (params: { window: typeof window }) => TReturn;
+
+type SafelyRunOnBrowserOptions<TReturn> = {
+  /**
+   * Fallback to run on server environments.
+   */
+  fallback: () => TReturn;
+};
+
+/**
+ * Safely runs code meant for browser environments only.
+ */
+export function safelyRunOnBrowser<TReturn>(
```

Comment:
I see why it's the same as the InstantSearch one, but maybe here it doesn't need the fallback? Either way fine for me though

Reply:
You're right, I changed it to return `undefined` in this case and we can add an actual fallback when we need it. See [b573a3d](https://github.com/algolia/autocomplete/pull/1106/commits/b573a3df658b5db5ce5b4b635d181870f66b327e).